src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/helper/DocumentHelper.java

```diff
@@ -164,30 +170,43 @@ public CaseDocument convertToCaseDocument(Object object) {
         return Optional.empty();
     }
 
-    public CaseDetails prepareLetterToApplicantTemplateData(CaseDetails caseDetails) {
+    public CaseDetails prepareLetterToPartyTemplateData(CaseDetails caseDetails, String partyToSendLetterTo) {
```

Comment:
You should never use strings to control logic, very bad practice. If the logic is different, have different methods